tests/test_utils/test_remote_user_api_auth_backend.py

```diff
@@ -1,95 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
-
-import unittest
-from unittest import mock
-
-from flask_login import current_user
-
-from airflow.www.app import create_app
-from tests.test_utils.config import conf_vars
-from tests.test_utils.db import clear_db_pools
-
-
-class TestGoogleOpenID(unittest.TestCase):
-    def setUp(self) -> None:
-        with conf_vars(
-            {("api", "auth_backend"): "tests.test_utils.remote_user_api_auth_backend"}
```

Comment:
The purpose of this test file appeared to be to test `tests.test_utils.remote_user_api_auth_backend`,  but that is "exercised" by the other Connexion tests which use this API, so we don't need to test this again.